### PR TITLE
PR: Add tutorial videos to help menu, update link to troubleshooting in docs

### LINKED ISSUE #13239: Add Tutorial Videos item to Spyder Help menu
## Problem Description

Now that we have our tutorial video series, we should consider adding a link to the playlist as an item in the Help menu, like we have the Spyder website, docs, Troubleshooting Guide, Google Group, and various other doc sites. Maybe not right away, but hopefully for 4.2 when they are more complete.

### PR DESCRIPTION
**Pull request by juanis2112**
## Description of Changes

* [x] Included a screenshot or animation (if affecting the UI, see [Licecap](https://www.cockos.com/licecap/))

- Added tutorial videos to help menu pointing to the playlist of spyder tutorials in Youtube 
- Removed Spyder tutorial from help menu as it is already in the help pane (and it will be removed in the future to be integrated with the docs)
- Updated link of troubleshooting guide in help menu to point to the new troubleshooting guide in docs
- Added new icon for video tutorials in help menu

Screenshot 2020-09-30 at 10 56 44 AM

Screenshot 2020-09-30 at 12 58 55 PM

### Issue(s) Resolved

Fixes #13239 

### Affirmation

By submitting this Pull Request or typing my (user)name below,
I affirm the [Developer Certificate of Origin](https://developercertificate.org)
with respect to all commits and content included in this PR,
and understand I am releasing the same under Spyder's MIT (Expat) license.

I certify the above statement is true and correct:

Juanis2112

### CODE REVIEW

File: spyder/app/mainwindow.py [hunk 1]
```diff
@@ -1049,11 +1049,12 @@ def create_edit_action(text, tr_text, icon):
...
                                "spyder documentation")
 
-        if self.help is not None:
-            tut_action = create_action(self, _("Spyder tutorial"),
-                                       triggered=self.help.show_tutorial)
-        else:
-            tut_action = None
+        spyder_vid = ("https://www.youtube.com/playlist"
+                      "?list=PLPonohdiDqg9epClEcXoAPUiK0pN5eRoc")
+        vid_action = create_action(self, _("Tutorial Videos"),
```
Comment (CAM-Gerlach) [inline spyder/app/mainwindow.py:1054]:
```suggestion
        vid_action = create_action(self, _("Tutorial videos"),
```

For consistency, this should use sentence case, as does every other action in this menu (besides, inexplicably, `Shortcut Summary`) and (most...) other places in Spyder.

File: spyder/app/mainwindow.py [hunk 2]
```diff
@@ -1049,11 +1049,12 @@ def create_edit_action(text, tr_text, icon):
...
+                                   icon=ima.icon('DialogHelpButton'),
```
Comment (CAM-Gerlach) [inline spyder/app/mainwindow.py:1055]: This icon repeats the one above, and implies troubleshooting or "rescueing" a user vs. teaching them new things (as tutorial videos are indented to), so it should either be replaced with something more appropriate, like a video icon from QtAwesome, or removed.

**Review by CAM-Gerlach — CHANGES_REQUESTED** [review]
Thanks @juanis2112 , excellent work. I tested this and everything worked and looked great, aside from two minor comments.

Also, don't forget to fixup your two cleanup commits into your first one before merging, to avoid trivial commits with non-descriptive, duplicate messages getting into the permanent history. Thanks!

File: spyder/utils/icon_manager.py [hunk 3]
```diff
@@ -240,6 +240,7 @@
     'DialogCloseButton':       [('fa.close',), {'color': MAIN_FG_COLOR}],
     'DirClosedIcon':           [('fa.folder-o',), {'color': MAIN_FG_COLOR}],
     'DialogHelpButton':        [('fa.life-ring',), {'color': 'darkred'}],
+    'TutorialVideosButton':    [('fa5s.video',), {'color': MAIN_FG_COLOR}],
```
Comment (ccordoba12) [inline spyder/utils/icon_manager.py:243]: Let's call this `VideoIcon` to make it a bit more generic (in case we want to use in other places later on).

**Review by ccordoba12 — COMMENTED** [review]
Thanks @juanis2112 for this! I left a small comment for you, otherwise looks good.

**Review by ccordoba12 — APPROVED** [review]
Thanks @juanis2112, looks good to me now.

**Review by CAM-Gerlach — COMMENTED** [review]
For what its worth, I was already in the process of reviewing as soon as the tests passed, and my review was dismissed and it was merged before I had the chance, but after testing this, this looks and works well so approve from me, thanks.

**Comment by ccordoba12** [comment]
Sorry, since your previous comments were already addressed, I thought you didn't have more to say. I'll wait for you for next time.

**Comment by CAM-Gerlach** [comment]
No problem, they were; sorry if I sounded more peeved than I was. I just wanted to give it one last check on Windows after unit tests were green since each platform has some differences in how icons are rendered, which ended up an issue while testing the new video icon we initially chose (which first crashed Spyder on Windows and Linux, then didn't look as we expected). Thanks!

**Pull request merged**